### PR TITLE
Decepticode

### PR DESCRIPTION
**Pull request by SENya1990**

### CODE REVIEW

File: src/Acuminator/Acuminator.Tests/Tests/Refactorings/ChangeEventHandlerSignatureToGeneric/ChangeEventHandlerSignatureToGenericTests.cs [hunk 1]
```diff
@@ -13,64 +13,64 @@
 
 namespace Acuminator.Tests.Tests.Refactorings.ChangeEventHandlerSignatureToGeneric
```
Comment (metallmonster) [inline src/Acuminator/Acuminator.Tests/Tests/Refactorings/ChangeEventHandlerSignatureToGeneric/ChangeEventHandlerSignatureToGenericTests.cs]: @SENya1990 do we want to merge this file with commented tests?

Comment (SENya1990) [reply src/Acuminator/Acuminator.Tests/Tests/Refactorings/ChangeEventHandlerSignatureToGeneric/ChangeEventHandlerSignatureToGenericTests.cs]: No, I missed it, good that you noticed

Comment (SENya1990) [reply src/Acuminator/Acuminator.Tests/Tests/Refactorings/ChangeEventHandlerSignatureToGeneric/ChangeEventHandlerSignatureToGenericTests.cs]: @metallmonster fixed, one more try

File: src/Acuminator/Acuminator.Vsix/source.extension.vsixmanifest [hunk 2]
```diff
@@ -1,32 +1,32 @@
...
+    </Dependencies>
+    <Assets>
+        <Asset Type="Microsoft.VisualStudio.MefComponent" d:Source="Project" d:ProjectName="Acuminator.Analyzers" Path="|Acuminator.Analyzers|" />
+        <Asset Type="Microsoft.VisualStudio.Analyzer" d:Source="Project" d:ProjectName="Acuminator.Analyzers" Path="|Acuminator.Analyzers|" />
+        <Asset Type="Microsoft.VisualStudio.MefComponent" d:Source="Project" d:ProjectName="%CurrentProject%" Path="|%CurrentProject%|" />
+        <Asset Type="Microsoft.VisualStudio.VsPackage" d:Source="Project" d:ProjectName="%CurrentProject%" Path="|%CurrentProject%;PkgdefProjectOutputGroup|" />
+        <Asset d:Source="Project" Type="Microsoft.VisualStudio.Assembly" d:ProjectName="Acuminator.Utilities" Path="|Acuminator.Utilities|" AssemblyName="|Acuminator.Utilities;AssemblyName|" />
+    </Assets>
+    <Prerequisites>
+        <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[15.0,16.0)" DisplayName="Visual Studio core editor" />
```
Comment (dropsonic) [inline src/Acuminator/Acuminator.Vsix/source.extension.vsixmanifest]: Does this mean that the next build will not support VS2015?

Comment (SENya1990) [reply src/Acuminator/Acuminator.Vsix/source.extension.vsixmanifest]: The same setting is in the master and dev Acuminator branches but it supports VS2015

**Pull request merged**